IssueTracker/IssueTracker.UnitTest/FileSystemTests/BlobDataTest.cs

```diff
@@ -0,0 +1,147 @@
+﻿using File = IssueTracker.FileSystem.Models.File;
+
+namespace IssueTracker.UnitTest.FileSystemTests;
+
+[TestClass]
+public class BlobDataTest : BaseClass
+{
+    [TestMethod]
+    [Description("Upload Method with Content null, i expected error")]
```

Comment:
Consistency!!!! Use the same format in all tests.

Reply:
<img width="452" alt="image" src="https://user-images.githubusercontent.com/74471664/190166378-7613fc1e-712f-4685-99ea-a4f205e7e3ee.png">